doc/expressionLanguage.md

```diff
@@ -552,7 +466,266 @@ Current common transformation set:
 | replaceallstr: (str, from, to)   | `str.replaceAll(from, to);`                                                                                             |
 | replaceallregexp: (str, reg, to) | `str.replaceAll(new RegExp(reg,"g"), to);`                                                                              |
 | split: (str, ch)                 | `str.split(ch);`                                                                                                        |
-| mapper: (val, values, choices)   | <code>choices[values.findIndex((target) &vert; target == val)]);</code>                                                  |
+| mapper: (val, values, choices)   | <code>choices[values.findIndex((target) &vert; target == val)]);</code>                                                 |
 | thmapper: (val, values, choices) | <code>choices[values.reduce((acc,curr,i,arr) &vert; (acc==0)&vert;&vert;acc?acc:val<=curr?acc=i:acc=null,null)];</code> |
 | bitwisemask: (i,mask,op,shf)     | <code>(op==="&"?parseInt(i)&mask: op==="&vert;"?parseInt(i)&vert;mask: op==="^"?parseInt(i)^mask:i)>>shf;</code>        |
-| slice: (arr, init, end)          | `arr.slice(init,end);`                                                                                                   |
+| slice: (arr, init, end)          | `arr.slice(init,end);`                                                                                                  |
+
+You have available this [JEXL interactive playground][99] with all the transformations already loaded, in which you can
+test all the functions described above.
+
+## Legacy Expression Language Transformations
+
+As described in previous sections, this is the default language just for backward compatibility reasons, but we strongly
+encourage you not to use this language for defining expression in order to transform measures, since it is not capable
```

Comment:
```suggestion
As described in previous sections, this is the default language just for backward compatibility reasons, but **we strongly
encourage you not to use this language** for defining expression in order to transform measures, since it is not capable
```

To remark it